IdentityCore/src/cache/MSIDUserInformation.m

```diff
@@ -0,0 +1,32 @@
+//
+//  MSIDUserInformation.m
+//  IdentityCore
+//
+//  Created by Sergey Demchenko on 12/7/17.
+//  Copyright © 2017 Microsoft. All rights reserved.
+//
+
+#import "MSIDUserInformation.h"
+
+@implementation MSIDUserInformation
```

Comment:
MSIDUserInformation will need backward compatibility class name as well.

Reply:
It is specified in MSIDKeyedArchiverSerializer.m

Reply:
I see, but do you need to set it each time separately or you could set it globally?